aiohttp/file_sender.py

```diff
@@ -152,16 +153,48 @@ def send(self, request, filepath):
         if not ct:
             ct = 'application/octet-stream'
 
-        resp = self._response_factory()
+        file_size = st.st_size
+
+        status = 200
+        start = 0
+        end = file_size - 1
+
+        # Handle 206 range response if requested
+        if 'range' in request.headers:
+            from .web_exceptions import HTTPPartialContent, HTTPRequestRangeNotSatisfiable
```

Comment:
I normally would by standard, but was following what had been done a few lines earlier in the same function. I'll fix both.